server/test/services/export/JsonExporterTest.java

```diff
@@ -163,6 +181,313 @@ private void testApplicationTopLevelAnswers(
+    resultAsserter.assertNullValueAtApplicationPath(".applicant_address.city");
+    resultAsserter.assertNullValueAtApplicationPath(".applicant_address.state");
+    resultAsserter.assertNullValueAtApplicationPath(".applicant_address.zip");
+  }
+
+  // CHECKBOX
+  @Test
+  public void export_whenCheckboxQuestionIsAnswered_valueIsInResponse() throws Exception {
+    createFakeQuestions();
+    createFakeProgram();
+    new FakeApplicationFiller()
+        .answerCheckboxQuestion(
+            ImmutableList.of(
+                2L, // "pepper grinder"
+                3L // "garlic press"
+                ))
+        .submit();
+
+    JsonExporter exporter = instanceOf(JsonExporter.class);
+
+    String resultJsonString =
+        exporter
+            .export(
+                fakeProgram.getProgramDefinition(),
+                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
+                SubmittedApplicationFilter.EMPTY)
+            .getLeft();
+    ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
+
+    resultAsserter.assertValueAtApplicationPath(
+        ".kitchen_tools.selections", ImmutableList.of("pepper grinder", "garlic press"));
+  }
+
+  @Test
+  public void export_whenCheckboxQuestionIsNotAnswered_valueInResponseIsEmptyArray()
+      throws Exception {
+    createFakeQuestions();
+    createFakeProgram();
+    new FakeApplicationFiller().submit();
+
+    JsonExporter exporter = instanceOf(JsonExporter.class);
+
+    String resultJsonString =
+        exporter
+            .export(
+                fakeProgram.getProgramDefinition(),
+                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
+                SubmittedApplicationFilter.EMPTY)
+            .getLeft();
+    ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
+
+    resultAsserter.assertValueAtApplicationPath(".kitchen_tools.selections", ImmutableList.of());
+  }
+
+  // CURRENCY
+  @Test
+  public void export_whenCurrencyQuestionIsAnswered_valueIsInResponse() throws Exception {
+    createFakeQuestions();
+    createFakeProgram();
+    new FakeApplicationFiller().answerCurrencyQuestion("5,444.33").submit();
+
+    JsonExporter exporter = instanceOf(JsonExporter.class);
+
+    String resultJsonString =
+        exporter
+            .export(
+                fakeProgram.getProgramDefinition(),
+                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
+                SubmittedApplicationFilter.EMPTY)
+            .getLeft();
+    ResultAsserter resultAsserter = new ResultAsserter(resultJsonString);
+
+    resultAsserter.assertValueAtApplicationPath(
+        ".applicant_monthly_income.currency_dollars", 5444.33);
+  }
+
+  @Test
+  public void export_whenCurrencyQuestionIsNotAnswered_valueInResponseIsNull() throws Exception {
+    createFakeQuestions();
+    createFakeProgram();
+    new FakeApplicationFiller().submit();
```

Comment:
Can the repeated prep work and injection be in a `@Before` method?

Reply:
The prep work is going to start to differ for each test with #5257, but I'll keep this in mind as part of that.

(In general I tend toward tests being upfront with their setup and asserts, so it's easy to see what's happening, vs hiding stuff in helper methods and `@Before` methods.)